src/core/Tokenizer.js

```diff
@@ -72,18 +99,19 @@ export default class Tokenizer {
         const patterns = {
             "``": "((`[^`]*($|`))+)",
             "[]": "((\\[[^\\]]*($|\\]))(\\][^\\]]*($|\\]))*)",
-            "\"\"": "((\"[^\"\\\\]*(?:\\\\.[^\"\\\\]*)*(\"|$))+)",
+            '""': '(("[^"\\\\]*(?:\\\\.[^"\\\\]*)*("|$))+)',
             "''": "(('[^'\\\\]*(?:\\\\.[^'\\\\]*)*('|$))+)",
             "N''": "((N'[^N'\\\\]*(?:\\\\.[^N'\\\\]*)*('|$))+)",
             "{{}}": "({{(.*?)}})",
+            "${}": "(\\${(.*?)})",
```

Comment:
Is it the main change? (difficult to see)

Reply:
Yes, I revert the styling stuff. Sorry about that